x-pack/plugins/security_solution/server/endpoint/mocks.ts

```diff
@@ -153,6 +154,9 @@ export const createMockPackageService = (): jest.Mocked<PackageService> => {
  */
 export const createMockFleetStartContract = (indexPattern: string): FleetStartContract => {
   return {
+    authz: {
```

Comment:
Thank you for the update here. At some point, we'll refactor this to actually use the Mocks out of Fleet instead of having our own.